lib/lightning_web/live/workflow_live/job_view.ex

```diff
@@ -183,6 +192,29 @@ defmodule LightningWeb.WorkflowLive.JobView do
     """
   end
 
+  def snapshot_version_chip(assigns) do
+    ~H"""
+    <div id="modal-header-workflow-snapshot-block" class="flex items-baseline">
+      <span
+        id="workflow-snapshot-version-chip"
+        phx-hook="Tooltip"
+        data-placement="bottom"
+        aria-label={
+          if @version == "latest",
+            do: "This is the latest version of the workflow.",
+            else:
+              "You are viewing a snapshot of this workflow that was made on #{Helpers.format_date(@snapshot_date)}"
+        }
+        class={"inline-flex items-center rounded-md bg-#{if @version == "latest", do: "blue-100", else: "yellow-100"} px-2 py-1 text-xs font-medium text-#{if @version == "latest", do: "blue-800", else: "yellow-800"}"}
```

Comment:
Tailwind can't detect partial/computed styles, so this should be:

`#{if @version == "latest", do: "bg-blue-100", else: "bg-yellow-100"}`

It may have worked by coincidence because the style was generated somewhere else.